go_faas_integration_test.go

```diff
@@ -0,0 +1,150 @@
+// +build integration
+
+package go_faas
+
+import "github.com/stretchr/testify/assert"
+
+func (suite *GoFaasTestSuite) TestFunctionsCRUD() {
+	funcName := "integration_testnodeinfo"
+
+	// get functions
```

Comment:
May be adding a `t.Log()` would help to understand the list of tests being executed.